KomaMRIPlots/Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.7.8"
 [deps]
 Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
 KomaMRICore = "4baa4f4d-2ae9-40db-8331-a7d1080e3f4e"
+KomaMRIIO = "fcf631a6-1c7e-4e88-9e64-b8888386d9dc"
```

Comment:
This should not be used in Plots

Reply:
Addressed in [094243f](https://github.com/cncastillo/KomaMRI.jl/commit/094243fce0d39a12721ea274223dfffd8cf1b6cc)